lib/lobbyist/v2/member_insights.rb

```diff
@@ -0,0 +1,12 @@
+module Lobbyist
+  module V2
+    class MemberInsights < Lobbyist::V2::HashieBase
+      attr_accessor :id
+
+      def self.insights(company_id, params = {})
```

Comment:
@sachinswagh-opcito 
1)make changes as per the api to the method name
2) remove the other unused  Lobbyist::V2::Company.insights

Reply:
@sreevs: 
1) Ok.
2) Lobbyist::V2::Company.insights is being used on results page. I think that is not in scope.